### DISCUSSION TITLE
Add Spoiler to the preview image or video in chat

### DISCUSSION THREAD

**Discussion by ThePharaohArt** [element-hq/element-meta#554] [Ideas]
#### What would you like to do?
I would like to add a spoiler or blur to a preview of an image or video until a user clicks it, similar to how the /spoiler command is done with text. 
#### Why would you like to do it?
Would be good to blur it if the images may be graphic in anyway, and perhaps some users wouldn't like to see it, so blurring the image or video would help "censor" the image until  click on it.
#### How would you like to achieve it?
Adding a spoiler button when uploading an image or video, which would then blur the preview

### Have you considered any alternatives?

Was not able to find any alternatives to do this.

**Comment by ThePharaohArt** [comment]
Still would like to see this implemented

**Comment by ChildishGiant** [comment]
Signal is also missing this feature but telegram and discord have it. Would be nice for there to be an encrypted messaging app with this feature
